pkg/component/rsyslogrelpconfigcleaner/rsyslog_relp_config_cleaner.go

```diff
@@ -177,6 +177,10 @@ func computeCommand() []string {
   chroot /host /bin/bash -c 'systemctl disable rsyslog-configurator; systemctl stop rsyslog-configurator; rm -f /etc/systemd/system/rsyslog-configurator.service'
 fi
 
+if [[ -d /host/var/log/rsyslog ]]; then
+  rm -rf /host/var/log/rsyslog
+fi
```

Comment:
```suggestion
rm -rf /host/var/log/rsyslog
```
```
$ rm --help | grep -e '--force'
  -f, --force           ignore nonexistent files and arguments, never prompt
```
Same for `test/integration/controller/lifecycle/lifecycle_test.go`

Reply:
This was to avoid the overhead of calling `rm` or `mkdir` multiple times since the script is run relatively often.

Reply:
```
$ time rm -rf foo

real	0m0.007s
user	0m0.002s
sys	0m0.003s

$ time mkdir -p go

real	0m0.018s
user	0m0.002s
sys	0m0.003s
```

We are in a nitpick territory between:
+2 lines and +0.02s speedup VS -2 lines and -0.02s speedup

So I'll give a lgtm and leave it up to you if you want to add my proposal